config/authors/skydoves.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Jaewoong Eum",
-  "bio": "Android and open source software engineer.❤️ Digital Nomad. Love coffee, music, magic tricks, and writing poems. Coffee Driven Development",
+  "bio": "Android and open source software engineer. Digital Nomad. Love coffee, music, magic tricks, and writing poems. Coffee Driven Development",
```

Comment:
Senior Android Developer Advocate @GetStream • GDE for Android & Kotlin.
Open Source Software Engineer